sql/src/test/java/ddlog/WeaveTest.java

```diff
@@ -53,7 +54,7 @@ public void testWeave() {
                 "  ephemeral_storage_allocatable bigint not null,\n" +
                 "  pods_allocatable bigint not null\n" +
                 ")";
-        DDlogIRNode create = t.translateSqlStatement(node_info);
+        DDlogIRNode create = t.translateSqlStatement(new PrestoSqlStatement(node_info));
```

Comment:
you could also have a local method translatePrestoSqlStatement that does these two things.